MacrobenchmarkSample/app/src/main/java/com/example/macrobenchmark/target/TrivialStartupActivity.kt

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
I suggested removing this before to minimize scope of sample - what's the reason for leaving it in?

Think it may be useful to compare the two UIs and observe the different performance?

Reply:
Yes, i thought it might be an easier to observe difference in performance.